go/ephemeral/user_ek_test.go

```diff
@@ -0,0 +1,44 @@
+package ephemeral
```

Comment:
Good point on these renames. Not sure what I was thinking :p

Reply:
all good!